### DISCUSSION TITLE
control the text color of TreeNode under cursor

### DISCUSSION THREAD

**Discussion by Fmajor** [Textualize/textual#4406] [Q&A] [answered]
I generate a tree and set node color by adding rich syntax in label, like `'[green] T async-thread[/]'`, and get the below tree
![image](https://github.com/Textualize/textual/assets/1620897/17db37fc-6c0a-4840-8810-984e4a4e6259)

but when I move the cursor to one tree, the text color is change to black, like this (the red color change to black)
![image](https://github.com/Textualize/textual/assets/1620897/386873d3-e8b8-41e9-881e-0f3ffe47ba10)

How can i set the cursor to only add background and do not change the text color?

I have try to change all settings in Tree.DEFAULT_CSS but with no luck
do we have something like `color: none` as in CSS3?
```
# I have try to change all settings in Tree.DEFAULT_CSS, I want some like color: none
    Tree {
        background: $panel;
        color: $text;
    }
    Tree > .tree--label {

    }
    Tree > .tree--guides {
        color: $success-darken-3;
    }

    Tree > .tree--guides-hover {
        color: $success;
        text-style: bold;
    }

    Tree > .tree--guides-selected {
        color: $warning;
        text-style: bold;
    }

    Tree > .tree--cursor {
        background: $secondary-darken-2;
        color: $text;
        text-style: bold;
    }

    Tree:focus > .tree--cursor {
        background: $secondary;
    }

    Tree > .tree--highlight {
        text-style: underline;
    }

    Tree > .tree--highlight-line {
        background: $boost;
    }
```

**Comment by TomJGooding** [comment, marked as answer]
I don't think you'll solve this with CSS unfortunately. I've not dived too deep into this but I suspect this is due to how the styling is applied in the code somewhere. There's an outstanding issue for how console markup is treated generally here:

- #3421
